### ISSUE TITLE
Add this mode to upstream CodeMirror directly

### ISSUE THREAD

**Issue by vassilevsky** [ianwalter/codemirror-mode-elixir#4]
Dear Ian,

I was wondering why you haven't submitted it to CodeMirror itself.

The reason I ask is that I'm using CodeMirror indirectly. I'm using Atlassian Bitbucket Server (formerly Stash) at work. It embeds CodeMirror and uses it to display all code, including pull requests.

My job is to review pull requests, and we code in Elixir (yay!), so it would have been terrific to have Elixir support there. But Atlassian embed vanilla CodeMirror and there is no Elixir support there :(

There is an issue about it in their issue tracker:

https://jira.atlassian.com/browse/BSERV-10181

I can help you with anything related.

Can you (we) do something to help it?

Thanks! 🙏

**Comment by ianwalter** [comment]
Hi Ilya, I created this because I had the same problem with pasting code snippets in Slack. I spoke with the creator of CodeMirror and he said they are not accepting new modes to core so I created this plugin and he linked to it from codemirror.net. I reached out to Slack to use it since its free and open source but I guess it wasn’t a high enough priority. I am actually in the middle of cleaning up the docs/site right now but when I’m done I’m definitely up to hassle all the big software products that use CodeMirror to support Elixir 😄
